src/generic/generic-component-list-item/generic-component-list-item.twig

```diff
@@ -70,11 +70,13 @@
       </div>
     {% endif %}
     <div class="ecl-list-item__body">
-      <div class="ecl-list-item__meta">
-        {% block meta_block %}
-          {% include '@ecl/generic-component-meta' with { metas: _metas } only %}
-        {% endblock %}
-      </div>
+      {% if _variant != 'navigation' %}
```

Comment:
Why not ` {% if _variant == 'thumbnail' %} ` ?

Reply:
but I see the `default` variant also uses metas, not sure about the rest but navigation will never need meta so I used "all but navigation" test

Reply:
what about the `default` variant?